### ISSUE TITLE
New Rule: no-debug

### ISSUE THREAD

**Issue by Turbo87** [simplabs/eslint-plugin-ember-concurrency#5]
http://ember-concurrency.com/docs/testing-debugging describes a `.debug()` task modifier at the bottom of the page. This new rule should warn about usage of this modifier to prevent it from being shipped to production.

### Bad 

```js
submitTask: task(function*() {
  yield this.post.save();
}).debug()
```

### Good 

```js
submitTask: task(function*() {
  yield this.post.save();
})
```
